context/app/static/js/components/detailPage/BulkDataTransfer/style.js

```diff
@@ -75,13 +80,13 @@ const StyledWarningIcon = styled(WarningRoundedIcon)`
   color: ${(props) => props.theme.palette.warning.main};
 `;
 
-const WarningIconContainer = styled.div`
+const IconContainer = styled.div`
   float: left;
   margin-right: ${(props) => props.theme.spacing(1)};
 `;
 
 const NoAccessContainer = styled.div`
-  display: flex;
+  display: ${(props) => (props.status === 'exclamation' ? 'block' : 'flex')};
```

Comment:
This would be a bit easier to follow if this ternary statement were in the `NoAccess` component and this prop was just the value being provided to `display`

Reply:
ah gotcha, made the change!